cypress/utils/random.ts

```diff
@@ -0,0 +1,11 @@
+
+
+export const generateRandomString = (length: number) => {
+    const charset = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789';
+    let password = '';
+    for (let i = 0; i < length; i++) {
+        const randomIndex = Math.floor(Math.random() * charset.length);
+        password += charset[randomIndex];
+    }
+    return password;
```

Comment:
@karenn32 Btw, variable must have the different name, since used not only for the password.